fs-tep-search/src/main/java/com/cgi/eoss/fstep/search/ipt/IptSearchProvider.java

```diff
@@ -200,11 +202,10 @@ private void addFstepProperties(Feature feature, SearchParameters parameters) {
 
         feature.setProperty("extraParams", extraParams);
 
-        HttpUrl.Builder quicklookUrlBuilder = parameters.getRequestUrl().newBuilder();
-        parameters.getRequestUrl().queryParameterNames().forEach(quicklookUrlBuilder::removeAllQueryParameters);
-        quicklookUrlBuilder.addPathSegment("ql").addPathSegment(productSource).addPathSegment(productIdentifier);
-        featureLinks.add(new Link(quicklookUrlBuilder.build().toString(), "quicklook"));
-
+        Object quicklookURL = extraParams.get("thumbnail");
+        if (quicklookURL != null) {
+        		featureLinks.add(new Link(quicklookURL.toString(), "quicklook"));
+        }
```

Comment:
The reason for the former approach was to allow us to create a local quicklooks cache. For IPT the thumbnail URL is directly accessible, but for other providers we might not be able to expose this to the user.

It's not an issue right now of course.